package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "pokemon-showdown-enhanced-tooltips",
+  "version": "0.1.0",
+  "description": "Enhanced tooltips for Pokemon Showdown in Chrome and Firefox",
+  "scripts": {
+    "build": "webpack"
+  },
+  "devDependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^6.0.3",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.12"
+  }
+}
```

Comment:
Webpack seems like the most popular way to do this sort of build process. If you'd prefer Grunt or Gulp, we could do that, but I have worked with Webpack most.

Reply:
I prefer webpack, too and I think there are advantages to choosing the most popular way to do the build process 👍